05-tidyeval.Rmd

```diff
@@ -0,0 +1,31 @@
+# Tidyeval
+
+## Spacing
+
+Put a space after the bang operators (`!!` and `!!!`) but not
+before them. Do not confuse `!!` and `!!!` with`!`.
+
+```{r, eval = FALSE}
+# Good
+call(!! xyz)
+
+# Bad
+call( !! xyz)
+call(! !xyz)
+```
+
+## Operator precedence
+
+Since all arithmetic and comparison operators bind more tightly than bang
+operators, one should be explicit about operator precedence. Enclose the
+expression to unquote in parenthesis or use `UQ()` and friends if arithmetic and
+comparison operators are involved.
+
+```{r, eval = FALSE}
+# Bad
+quo(1 + !! 1 + 2 + 3)
+
+# Good
+quo(1 + (1 + 2 + 3))
```

Comment:
I think you're missing !! here

And I think for now, let's not promote the use of UQ.